akka-sample-distributed-data-scala/src/main/scala/sample/distributeddata/ShoppingORCart.scala

```diff
@@ -0,0 +1,143 @@
+package sample.distributeddata
+
+import java.util.UUID
+
+import scala.concurrent.duration._
+import akka.actor.Actor
+import akka.actor.ActorRef
+import akka.actor.Props
+import akka.cluster.Cluster
+import akka.cluster.ddata.{DistributedData, ORMultiMap, ORMultiMapKey}
+
+import scala.collection.immutable.HashSet
+
+object ShoppingORCart {
+  import akka.cluster.ddata.Replicator._
+
+  def props(userId: String): Props = Props(new ShoppingORCart(userId))
+
+  case object GetCart
+  final case class ChangeItemQuantity(item: LineItem)
+  final case class RemoveItem(productId: String)
+
+  final case class Cart(items: Set[LineItem])
+  final case class LineItem(productId: String, title: String, quantity: Int)
+  final case class SingleLineItem(productId: String, title: String, uid: String)
+
+  //#read-write-majority-orcart
+  private val timeout = 3.seconds
+  private val readMajority = ReadMajority(timeout)
+  private val writeMajority = WriteMajority(timeout)
+  //#read-write-majority-orcart
```

Comment:
There aren't any docs pulling this in right?

Reply:
No. What docs are using akka-samples code? I'd be happy to contribute some doc for this to accompany the other ones.

Reply:
I don't think we have any, I think the scaladoc above will be an improvement, please remove these doc-markups to avoid future confusion :)